.github/workflows/stop-dev-environment.yml

```
@@ -30,4 +30,40 @@ jobs:
           aws-region: ${{ secrets.AWS_REGION }}
           aws-role: ${{ secrets.AWS_ASSUME_ROLE }}
           eks-cluster: ${{ secrets.EKS_CLUSTER_NAME }}
-          environment-name: 'dev'
\ No newline at end of file
+          environment-name: 'dev'
```

actionlint (1.7.4)

input "environment-name" is not defined in action "Stop EKS Environment" defined at "./.github/actions/stop-dev". available inputs are "aws-region", "aws-role", "eks-cluster"

(action)